recipe/meta.yaml

```diff
@@ -24,9 +21,12 @@ build:
     - export PETSC_DIR=$PREFIX
     - {{ PYTHON }} -m pip -v install --no-deps .
   skip: true  # [win]
-  string: {{ scalar }}_h{{ PKG_HASH }}_{{ build }}
+  skip: true  # [osx and device != "host"]
```

Comment:
this skip can be replaced with selectors in conda_build_config above

Reply:
go ahead and make the changes